src/main/java/duke/Duke.java

```diff
@@ -0,0 +1,41 @@
+package duke;
```

Comment:
I like how you extracted out the classes and packages, it makes the overall structure of your code really neat!

Reply:
Thanks! 😄